src/proof_system/prover.rs

```diff
@@ -145,6 +147,30 @@ impl Prover {
         self.preprocessed_transcript.append_message(label, message);
     }
 
+    /// Adds the blinding scalars to a given vector. Always the same elements
+    /// of 'w_vec' are modified at the beginning of it, and appended at the end:
+    /// if hiding degree = 1: (b2*X(n+1) + b1*X^n - b2*X - b1) + w_vec
+    /// if hiding degree = 2: (b3*X^(n+2) + b2*X(n+1) + b1*X^n - b3*X^2 - b2*X
+    /// - b1) + w_vec
+    pub(crate) fn blind_poly<R: RngCore + CryptoRng>(
+        w_vec: &Vec<dusk_bls12_381::BlsScalar>,
+        hiding_degree: usize,
+        domain: &EvaluationDomain,
+        mut rng: &mut R,
+    ) -> Polynomial {
+        let mut w_vec_i = domain.ifft(w_vec);
```

Comment:
I find the names `w_vec` and `w_vec_i` not so descriptive. Isn't `w_vec_i` just `w_poly` and `w_vec` the poly in evaluation form? What about calling them `w_poly` and `w_poly_evals`?

Reply:
I just followed the same naming used in the whole code, we should rename the whole code if we want to change it here... And I used `w_vec_i` where `i` comes from `inverse`. I used this name the distinguish the value we get before calling `Polynomial::from_coefficients_vec()`

Reply:
Oh i see, then please call it `w_vec_inverse`. At first sight I thought that you used the `i` because you want to loop through the coefficients later on